test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -884,6 +886,32 @@ void UcxConnection::set_log_prefix(const struct sockaddr* saddr,
     memcpy(_log_prefix, ss.str().c_str(), length);
 }
 
+void UcxConnection::print_addresses()
+{
+    ucp_ep_attr_t ep_attr;
+
+    if (_ep == NULL) {
+        return;
+    }
```

Comment:
why needed?

Reply:
@yosefe If _ep is NULL, calling of ucp_ep_query(_ep, &ep_attr) will have segment fault.